tests/console/journalctl.pm

```diff
@@ -185,10 +185,16 @@ sub run {
 
     assert_script_run("date '+%F %T' | tee /var/tmp/reboottime");
     assert_script_run("echo 'The batman is going to sleep' | systemd-cat -p info -t batman");
+    script_run('journalctl --list-boots');    # Debug output to help identify issues when less than 2 boots are displayed
     reboot($self);
+    assert_script_run('journalctl --flush');    # Ensure that the current boot is flushed to the journal
```

Comment:
Is it a product bug that the journal logs are not flushed after system reboot automatically?

Reply:
Good question. I try to reproduce the issue in a local VM and if successful file a bug report.

Reply:
See https://progress.opensuse.org/issues/122683#note-13, this flush is likely not needed. I removed it from the PR.